ParabolInc/parabol
# feat(jira-server): sprint poker vote to comment

Resolves #5811
<img width="922" alt="image" src="https://user-images.githubusercontent.com/466991/161594010-8e2e3b51-0212-4f58-859c-242d460f2b33.png">


Only ability to vote to comment in implemented for now.
Next issue #5812

How to test:

- Create a poker meeting
- Add some Jira Server issue
- Vote and update score
- See a comment appeared in Jira Server
